docs/get-started/going-live.md

```diff
@@ -49,22 +53,23 @@ However, if you want to use a different database, for example, you can create a
 From the root of your project, run the following command:
 
 ```
-webiny deploy-api --env=dev
+webiny deploy api --env=dev
 ```
 
+
 ## 3. Deploy apps using `dev` environment
 
 From the root of your project, run the following command:
 
 ```
-webiny deploy-apps --env=dev
+webiny deploy apps --env=dev
 ```
 
 ## Other environments
 
 If you also want to deploy a `prod` environment, repeat the procedure using the `prod` value.
 
-If you need more environments, you can easily add those following the same procedure, but you will need to also add the appropriate build scripts to your apps, in the following files:
+This can be extended if you need more environments, you can easily add them in the same procedure shown above, but you will need to also add the appropriate build scripts to your apps, in the following files:
```

Comment:
Please put the production environment deployment into its own section. For example `Deploying to prod`.

Then, let's use this content for the `Other environments` section:

## Other environments
Apart from seen `dev` and `production` environments, Webiny also supports adding new ones. In order to do so, the same steps described above are to be taken, with one additional step and that is inserting a build script for existing React apps, in their respective `package.json` files:

```
apps/admin/package.json
apps/site/package.json
apps/another-app/package.json
```

So, an example script for a `staging` environment could look like the following:

```
"build:staging": "env-cmd -r .env.json -e default,staging webiny run build"
```

Once this is inserted, as usual, the following command should successfully execute a `staging` environment deployment:

```
webiny deploy apps --env=staging
```

Reply:
Added